imls-frontend/src/views/PageLibrary.vue

```diff
@@ -36,23 +43,28 @@ export default {
     }
   },
   computed: {
-    activeDate() {
-      return this.selectedDate;
+    selectedDateUTC() {
```

Comment:
I decided to make sure it was clear that these dates were in UTC (not ISO or some other format) so they've been renamed